pkg/controller/plan/kubevirt.go

```diff
@@ -1584,6 +1585,29 @@ func (r *KubeVirt) guestConversionPod(vm *plan.VMStatus, vmVolumes []cnv.Volume,
 	if err != nil {
 		return
 	}
+	if vm.RootDisk != "" {
+		vmInventory := &model.VM{}
```

Comment:
since we might query the inventory also in other functions (like the `PodEnvironment`), maybe we can query it once at the beginning and send it to the function instead querying it multiple times?

Reply:
yes it's not an offical PR, just a draft we worked on last week, wanted to test it on the the customer version and share it with Benny, but will open this on the main once its ready.